server/database/quieres/getAllProducts.js

```diff
@@ -0,0 +1,5 @@
+const connection = require('../connection');
+
+const getAllProducts = () => connection.query('select id, name, img_url, rating, old_price, new_price from products ;');
```

Comment:
please add a product description, quantity, and category
and don't forget to modify your test.

Reply:
done